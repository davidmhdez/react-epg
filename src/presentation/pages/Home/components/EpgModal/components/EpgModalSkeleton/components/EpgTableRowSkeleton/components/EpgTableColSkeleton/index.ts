export {default} from "./EpgTableColSkeleton.tsx"