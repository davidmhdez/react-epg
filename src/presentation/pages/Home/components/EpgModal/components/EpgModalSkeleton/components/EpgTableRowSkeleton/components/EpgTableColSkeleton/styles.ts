import styled from "styled-components";

export const EpgTableColSkeletonWrapper = styled.div<{$width: number}>`
    width: ${({ $width }) => $width}px;
    height: 120px;
    border: 1px solid #6C6C6CAD;
    
    .programTitle{
        width: 80px;
        height: 20px;
        margin: 20px;
    }
    
    .programDuration{
        width: 100px;
        height: 15px;
        margin-left: 20px;
    }
`;