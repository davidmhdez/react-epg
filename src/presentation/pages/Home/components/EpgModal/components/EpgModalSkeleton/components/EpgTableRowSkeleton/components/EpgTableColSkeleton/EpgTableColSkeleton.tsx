import {EpgTableColSkeletonWrapper} from "./styles.ts";
import Skeleton from "react-loading-skeleton";

type Props = {
    width: number
}

const EpgTableColSkeleton: React.FC<Props> = ({width}) => {
    return (
        <EpgTableColSkeletonWrapper $width={width}>
            <Skeleton className="programTitle"/>
            <Skeleton className="programDuration"/>
        </EpgTableColSkeletonWrapper>
    );
};

export default EpgTableColSkeleton;
