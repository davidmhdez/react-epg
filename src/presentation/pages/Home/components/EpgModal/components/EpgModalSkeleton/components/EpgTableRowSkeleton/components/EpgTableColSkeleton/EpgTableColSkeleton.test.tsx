import EpgTableColSkeleton from "./EpgTableColSkeleton.tsx";
import {render} from "@testing-library/react";

describe('EpgTableColSkeleton test', () => {
    test('should match with snapshot', () => {
        const { container } = render(<EpgTableColSkeleton width={30}/>);
        expect(container).toMatchSnapshot()
    })
})