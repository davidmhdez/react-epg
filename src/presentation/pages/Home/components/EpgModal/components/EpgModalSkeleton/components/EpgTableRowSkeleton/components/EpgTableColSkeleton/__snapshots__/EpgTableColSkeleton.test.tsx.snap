// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EpgTableColSkeleton test should match with snapshot 1`] = `
<div>
  <div
    class="sc-aYaIB eHqskW"
  >
    <span
      aria-busy="true"
      aria-live="polite"
    >
      <span
        class="react-loading-skeleton programTitle"
      >
        ‌
      </span>
      <br />
    </span>
    <span
      aria-busy="true"
      aria-live="polite"
    >
      <span
        class="react-loading-skeleton programDuration"
      >
        ‌
      </span>
      <br />
    </span>
  </div>
</div>
`;
